modules/app/read_write.py

```diff
@@ -15,15 +16,19 @@ def __init__(self) -> None:
         self.textDir = self.dir.joinpath(self.settings.txt_subdir)
         self.pdfDir = self.dir.joinpath(self.settings.pdf_subdir)
 
+    class FilesDict(TypedDict):
+        filename: str
+        contents: bytes
+
     def getFiles(self, path: Path, count_only: bool = False) -> list:
```

Comment:
For the purpose of type hinting and checking right in the IDE?

Reply:
Basically, yeah; to tell editors what a list of class `FilesDict` contains, indeed, so it’s also easier to work with elements in such a list (when doing `for item in items`, for example). I think Python itself ignores things like this.